yasa-solr-plugin/src/main/java/io/github/kezhenxu94/YasaHandler.java

```diff
@@ -36,35 +42,49 @@
 import org.apache.solr.response.SolrQueryResponse;
 import org.apache.solr.security.PermissionNameProvider;
 
-import static io.github.kezhenxu94.YasaHandler.PLUGIN_PATH;
-
 @EndPoint(
   method = METHOD.GET,
-  path = PLUGIN_PATH + "/*",
+  path = "$path-prefix/*",
   permission = PermissionNameProvider.Name.CONFIG_READ_PERM
 )
 @RequiredArgsConstructor
-@SuppressWarnings("unused")
-public class YasaHandler {
-  public static final String PLUGIN_PATH = "/plugin/yasa";
-
+public class YasaHandler implements ResourceLoaderAware {
+  @SuppressWarnings("unused")
   private final CoreContainer coreContainer;
 
+  private ResourceLoader loader = null;
+
+  @Override
+  public void inform(ResourceLoader loader) throws IOException {
+    this.loader = loader;		
+  }
+
   @Command
```

Comment:
The `@EndPoint` annotation should go to the method here and please remove the `@Command` annotation

Reply:
It is compiled against Solr 8.4 right now, and it is working fine (even with Solr 8.6). Method level @Endpoint annotation is available only from Solr 8.6 onwards. Maybe we can make this change after 8.6 is released.

Reply:
Once 8.6 binaries are out we can make that change and release it

Reply:
Nothing stopping us from releasing right away. It is working fine with 8.6.

Reply:
Let keep it as is for now to make it work, and once the 8.6 is released, I (or anyone) can change it to accord the new API in 8.6 and release it

BTW, as YASA is not (yet) part of ASF, we can do the release in a very lite-weight manner, so we can do a release whenever we think it's needed